pure_pursuit_wrapper/include/pure_pursuit_wrapper/pure_pursuit_wrapper.hpp

```diff
@@ -36,14 +37,22 @@ class PurePursuitWrapper {
         PurePursuitWrapper(WaypointPub waypoint_pub, PluginDiscoveryPub plugin_discovery_pub);
 
         void trajectoryPlanHandler(const cav_msgs::TrajectoryPlan::ConstPtr& tp);
+        /**
+         * \brief callback function for jerk published by stop_and_wait_plugin. This function updates the local jerk variable used in time_to_speed calculation
+         * \param jerk float64 message with jerk value from stop_and_wait_plugin
+         */
+        void updatejerk(std_msgs::Float64 jerk);
 
         bool onSpin();
 
     private:
     WaypointPub waypoint_pub_;
     PluginDiscoveryPub plugin_discovery_pub_;
+    ros::Subscriber jerk_sub_;  //Jerk value from stop&wait plugin
```

Comment:
This could be used by any plugin, so let's make sure this solution works for those cases as well.

Reply:
This PR is specific to handling jerk calculation for stop&wait. This pr includes a separate wrapper for plugins needing constant jerk https://github.com/usdot-fhwa-stol/carma-platform/compare/feature/add_const_jerk_wrapper